main/lsp/requests/prepare_rename.cc

```diff
@@ -22,6 +34,32 @@ variant<JSONNullObject, unique_ptr<PrepareRenameResult>> getPrepareRenameResult(
     return result;
 }
 
+variant<JSONNullObject, unique_ptr<PrepareRenameResult>>
+getPrepareRenameResultForIdent(const core::GlobalState &gs, const core::lsp::IdentResponse *identResp) {
+    ENFORCE(identResp->termLoc.exists());
+
+    // The loc for the instance variable local in `attr_reader :foo`
```

Comment:
Do we want tests for `prop` and/or `const` as well?

Reply:
Can props be matched somehow? The issue with attributes was that they are connected to variables, which my trigger incorrect renames.